### PR TITLE
Fix molecules analyzer

### LINKED ISSUE #221: Accept single quotes in the molecule validation regex

### PR DESCRIPTION
**Pull request by FrancoisPog**
closes #221 

- [x] Accept single quotes in molecule names
- [x] Increase the maximum length of a property value
- [x] Update the data of molecules used with the new Marco spreadsheet

### CODE REVIEW

**Review by valentinperignon — APPROVED** [review]
insapinsaponifiable d’avocat et LGTM de sojaonifiable

File: server/test/database.test.js [hunk 1]
```diff
@@ -161,9 +161,8 @@ describe("Procedures Molecule data", () => {
       (e) => e.cl_name
     );
     expect(classes).to.be.deep.equals([
-      "camion",
-      "INTI (INHIBITEURS NUCLEOSIDIQUES TRANSCRIPTASE INVERSE)",
-      "ANALOGUES NUCLEOSIDIQUES",
+      "INTI (INHIBITEURS NUCLEOTIDIQUES TRANSCRIPTASE INVERSE)",
```
Comment (nathanaelhoun) [inline server/test/database.test.js:164]: No more :truck:? :sob:

**Review by nathanaelhoun — APPROVED** [review]
LGTM!

File: server/test/database.test.js (hunk 1 above)
Comment (FrancoisPog) [reply server/test/database.test.js:164]: I am so sorry ... Vroum Vroum 🚚

**Pull request merged**